idea_town/frontend/static-src/app/lib/page-manager.js

```diff
@@ -17,6 +19,8 @@ export default class PageManager {
       throw new Error('PageManager constructor must be passed a pageContainer element');
     }
 
+    this.header = document.querySelector('header');
```

Comment:
This selector seems a bit _generic_ and could possibly break if we had multiple `<header>` elements in the DOM, no?